.clang-format

```diff
@@ -22,7 +23,7 @@ BreakBeforeBinaryOperators: All
 BreakBeforeBraces: Attach
 BreakBeforeTernaryOperators: true
 BreakConstructorInitializersBeforeComma: true
-ColumnLimit:     96
+ColumnLimit: 80
```

Comment:
May want to remove these changes lint changes and put them in a preceding commit, for the sake of making the PR clearer.